src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -9192,6 +9193,121 @@ public class LibraryClient implements BackgroundResource {
     return stub.streamingArchiveBooksCallable();
   }
 
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
+  /**
+   * Test use resource message as request objects. Only used by PubSub (CreateSubscription) for historical reasons.
+   * New APIs should always create a separate message for a request.
+   *
+   * Sample code:
+   * <pre><code>
+   * try (LibraryClient libraryClient = LibraryClient.create()) {
+   *   BookName name = BookName.ofShelfBookName("[SHELF]", "[BOOK]");
+   *   String author = "";
+   *   String title = "";
+   *   Book.Rating rating = Book.Rating.GOOD;
+   *   libraryClient.saveBook(name, author, title, rating);
+   * }
+   * </code></pre>
+   *
+   * @param name The resource name of the book.
+   * Book names have the form `bookShelves/{shelf_id}/books/{book_id}`.
+   * Message field comment may include special characters: &lt;&gt;&amp;"`'{@literal @}.
+   * @param author The name of the book author.
+   * @param title The title of the book.
+   * @param rating For testing enums.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  public final void saveBook(BookName name, String author, String title, Book.Rating rating) {
```

Comment:
This is the only baseline change that matters... everything else is caused by adding an RPC.